ws/src/speech/scripts/STT_microservice/README.md

```diff
@@ -0,0 +1,20 @@
+# Speech To Text Microservice
+
+Contains two gRPC servers, one uses whisper and the other faster-whisper for improved speed and precision. To run with cuda on l4t a docker image was generated with CTranslate2. You can pull it by running:
+
+```bash
+docker pull tacolon27/faster-whisper-l4t-cuda
```

Comment:
Add instructions or Dockerfile to recreate image.

Reply:
I need access to xavier to do this, since it involves a lot of testing because of the CTranslate2 compilation from source which was a huge problem. Will add in a future PR if thats okay.